android/src/main/java/bz/rxla/flutter/speechrecognition/SpeechRecognitionPlugin.java

```diff
@@ -28,6 +28,7 @@
     private String transcription = "";
     private Intent recognizerIntent;
     private Activity activity;
+    private currentRMS = 2.0f;
```

Comment:
Why the default rms is set to `2.0`? Also the variable type is missing `double`.